stable/redmine/README.md

```diff
@@ -87,6 +87,11 @@ The following table lists the configurable parameters of the Redmine chart and t
 | `persistence.storageClass`        | PVC Storage Class                        | `nil` (uses alpha storage class annotation)             |
 | `persistence.accessMode`          | PVC Access Mode                          | `ReadWriteOnce`                                         |
 | `persistence.size`                | PVC Storage Request                      | `8Gi`                                                   |
+| `podDisruptionBudget.enabled`     | Pod Disruption Budget toggle             | `false`                                                 |
+| `podDisruptionBudget.minAvailable`| Minimum availble pods                   | `nil`                                                     |
```

Comment:
There's still a typo `availble` -> `available`
The same on the next line `unavailble` -> `unavailable`